configstack/module.go

```diff
@@ -116,25 +116,51 @@ func (module *TerraformModule) planFile(terragruntOptions *options.TerragruntOpt
 func (module *TerraformModule) outputFile(opts *options.TerragruntOptions) string {
 	planFile := ""
 
-	if opts.OutputFolder != "" {
-		path, _ := filepath.Rel(opts.WorkingDir, module.Path)
-		dir := filepath.Join(opts.OutputFolder, path)
-		planFile = filepath.Join(dir, terraform.TerraformPlanFile)
+	if opts.OutputFolder == "" {
+		return planFile
 	}
 
+	path, _ := filepath.Rel(opts.WorkingDir, module.Path)
+	dir := filepath.Join(opts.OutputFolder, path)
+
+	if !filepath.IsAbs(dir) {
+		dir = filepath.Join(opts.WorkingDir, dir)
+	}
+
+	if absDir, err := filepath.Abs(dir); err == nil {
```

Comment:
Not sure if this expression is needed since it will be absolute after the line 127.